app/views/dashboard.erb

```diff
@@ -6,7 +6,7 @@
         <% if dashboard.current_exercises.any? %>
             <p>Keep working on these problems, or archive them.</p>
             <ul class="nav nav-stacked" id="menu-current" role="menu">
-              <% dashboard.current_exercises.limit(5).each do |exercise| %>
+              <% dashboard.current_exercises.reverse.limit(5).each do |exercise| %>
```

Comment:
@esparta I considered the `limit(5).reverse` path as well, but that only helps the user for the first 5 exercises, as it still pulls the first 5 exercises and then reverses their order, while we need the pull the last 5 exercises here.

@Insti why does `by_activity` change to the last completed first, rather than continuing to pull the first completed first?